Uni_life_tracker/app/src/main/java/com/generic/ult/Add_expenses.java

```diff
@@ -0,0 +1,56 @@
+package com.generic.ult;
+
+import android.os.Bundle;
+import android.view.View;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.navigation.ui.AppBarConfiguration;
+import com.google.android.material.textfield.TextInputLayout;
+
+import java.util.Objects;
+
+
+public class Add_expenses extends AppCompatActivity {
```

Comment:
I think the name here should be camelCase too. Remember to refactor and rename to avoid problems with other dependences

Reply:
Fixed